src/runtime/virtcontainers/mount.go

```diff
@@ -226,6 +226,40 @@ func isDeviceMapper(major, minor int) (bool, error) {
 	return false, err
 }
 
+var blockFormatTemplates = map[string]string{
+	"dm":    "/sys/dev/block/%d:%d/dm",
+	"block": "/sys/dev/block/%d:%d/device/block",
+	"loop":  "/sys/dev/block/%d:%d/loop",
+}
+
+func isBlockDevice(major, minor int, formatTemplate string) (bool, error) {
+	sysPath := fmt.Sprintf(formatTemplate, major, minor)
+
+	_, err := os.Stat(sysPath)
+	if err == nil {
+		return true, nil
+	} else if os.IsNotExist(err) {
```

Comment:
I think we can remove the `else if`, or to add the last return into an `else` block for consistency

```
if err == nil {
    return true, nil
}

if os.IsNotExist(err) {
    return false, nil
}

return false, err
```

Reply:
done